ads/directadvert.md

```diff
@@ -0,0 +1,34 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Directadvert
+
+## Example
+
+```html
+<amp-ad width="320" height="250"
+    type="directadvert"
+    data-block-id="1953045">
+</amp-ad>
+
+```
+
+## Configuration
+
+For more information, please [see FAQ](https://www.directadvert.ru/text/help).
+
+Supported parameters:
```

Comment:
Please indicate this is required parameter.

Reply:
Fixed